public/javascripts/searchBoxHome.js

```diff
@@ -15,7 +15,7 @@ var searchBoxHome = function() {
                 }
             }, [
                 m('input.form-control[type=text]', {
-                    placeholder: 'Title, Lyric or Author',
+                    placeholder: 'Search by title, lyric or author',
```

Comment:
please use 'lyrics'
http://www.differencebetween.com/difference-between-lyric-and-vs-lyrics/